src/main/java/com/commercetools/emailprocessor/email/EmailProcessor.java

```diff
@@ -76,10 +78,13 @@ public EmailProcessor() {
                         statistics.update(0);
                     }
                 }
+                client.close();
                 return statistics;
-            }
-
-        );
+            })
+            .exceptionally(exception -> {
```

Comment:
Exception is swallowed?

Reply:
bfe95ed34eecc8d8ce404bbdb5e67df671c71d30